doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -454,933 +536,640 @@ includes considerations specific to deployments at the edge of the network. The
-|                                        |           | standard for business continuity.    |                         |
-|                                        |           | ISO/IEC 27031 - ISO/IEC 27031 is the |                         |
-|                                        |           | international Standard for ICT       |                         |
-|                                        |           | readiness for business continuity.   |                         |
-+----------------------------------------+-----------+--------------------------------------+-------------------------+
-|`7.9.14`                                |sec.std.012| The Public Cloud Operator **must**,  |                         |
-|                                        |           | and the Private Cloud Operator       |                         |
-|                                        |           | **may** be certified to be compliant |                         |
-|                                        |           | with the International Standard on   |                         |
-|                                        |           | Awareness Engagements (ISAE) 3402    |                         |
-|                                        |           | (in the US: SSAE 16). International  |                         |
-|                                        |           | Standard on Awareness Engagements    |                         |
-|                                        |           | (ISAE) 3402. US Equivalent: SSAE16.  |                         |
-+----------------------------------------+-----------+--------------------------------------+-------------------------+
+.. list-table:: Reference Model Requirements: Cloud Infrastructure Security Requirements
+   :widths: 10 10 70 10
+   :header-rows: 1
+
+   * - Reference Model Section
+     - Reference
+     - Description
+     - Specification Reference
+   * - `7.9.1`
+     - sec.gen.001
+     - The Platform **must** maintain the specified configuration.
+     -
+   * - `7.9.1`
+     - sec.gen.002
+     - All systems part of Cloud Infrastructure **must** support password hardening as defined in `CIS Password Policy Guide <https://www.cisecurity.org/white-papers/cis-policy-guide/>`__. Hardening: CIS Password Policy Guide
+     - `5.3.1 Node Hardening: Securing Kubernetes Hosts`
+   * - `7.9.1`
+     - sec.gen.003
+     - All servers part of Cloud Infrastructure **must** support a root of trust and secure boot.
+     -
+   * - `7.9.1`
+     - sec.gen.004
+     - The Operating Systems of all the servers part of Cloud Infrastructure **must** be hardened by removing or disabling unnecessary services, applications and network protocols, configuring operating system user authentication, configuring resource controls, installing and configuring additional security controls where needed, and testing the security of the Operating System. (NIST SP 800-123)
```

Comment:
Don't we need to linewrap to al least 120 chars even in this case?

Reply:
Yes